src/index.ts

```diff
@@ -19,18 +19,22 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-export type { AccessModes } from "./type/AccessModes";
 export type { AccessBaseOptions } from "./type/AccessBaseOptions";
+export type { AccessCredentialType } from "./type/AccessCredentialType";
+export type { AccessGrantContext } from "./type/AccessGrantContext";
+export type { AccessModes } from "./type/AccessModes";
+export type { ApproveAccessRequestOverrides } from "./manage/approveAccessRequest";
+export type { FetchOptions } from "./type/FetchOptions";
 export type { IssueAccessRequestParameters } from "./type/IssueAccessRequestParameters";
+export type { RedirectOptions } from "./type/RedirectOptions";
```

Comment:
Might be good to merge this type into `RedirectToAccessManagementUiOptions` rather than having it seperate?

Reply:
Sounds good, but maybe let's address this in a separate PR.

Reply:
sure!